nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nuxeo.defaults

```diff
@@ -0,0 +1,17 @@
+## DO NOT EDIT THIS FILE, USE nuxeo.conf  ##
+nuxeo.template.includes=s3binaries
+#GLACIER
+nuxeo.glacier.target=.
+
+# mandatory
+nuxeo.aws.glacier.bucket=
+nuxeo.aws.glacier.bucket_prefix=
```

Comment:
do we thing the `bucket_prefix` will be mandatory.

Reply:
no, will be moved into the optional section